.eslintrc.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": ["./packages/internal-tooling/eslint/index.js"],
```

Comment:
Paths are not great, should be fixed in ESLint 6: https://github.com/eslint/eslint/pull/11388